src/System.Runtime.Caching.Analyzers/AnalyzerReleases.Shipped.md

```
@@ -0,0 +1,2 @@
+ ; Shipped analyzer releases
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)